Planner/qSlicerPlannerModuleWidget.cxx

```diff
@@ -60,6 +60,7 @@
 #include "vtkMRMLMarkupsDisplayNode.h"
 #include "vtkMRMLMarkupsPlaneNode.h"
 #include "vtkMRMLSubjectHierarchyNode.h"
+#include "vtkMRMLHierarchyNode.h"
```

Comment:
@sjh26 @bpaniagua 

I don't think I should need vtkMRMLHierarchyNode.h for anything. I propose removing this.

Reply:
I implemented this changes in https://github.com/KitwareMedical/OsteotomyPlanner/pull/118/commits/7b61b08bcf522ddc43c569a8bde5465f6d376aca.